Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "crossbeam-deque"
+version = "0.1.0"
+authors = ["The Crossbeam Project Developers"]
+license = "MIT/Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/crossbeam-rs/crossbeam-deque"
+homepage = "https://github.com/crossbeam-rs/crossbeam-deque"
+documentation = "https://docs.rs/crossbeam-deque"
+description = "Concurrent work-stealing deque"
+keywords = ["Chase-Lev", "lock-free", "scheduler", "scheduling"]
+categories = ["concurrency", "data-structures"]
+
+[badges]
+travis-ci = { repository = "crossbeam-rs/crossbeam-deque" }
+
+[dependencies]
+crossbeam-epoch = { git = "https://github.com/stjepang/crossbeam-epoch", branch = "guard" }
+crossbeam-utils = { git = "https://github.com/crossbeam-rs/crossbeam-utils" }
```

Comment:
We published crossbeam-utils in crates.io.  What about using it?

Reply:
Yeah, I should just publish a new version... The latest version was published a while ago.